README-local-package-graph.md

```diff
@@ -0,0 +1,16 @@
+## To make a Mermaid graph for your operator index
+
+### Required:
+sed, make, git, container execution environment (such as Docker, podman)
+
+### Usage
+```bash
+git clone git@github.com:jmccormick2001/sample-operator
+cd sample-operator
+make bundle-build
+docker tag controller-bundle:2.0.0 quay.io/btofel/controller-bundle:v2.0.0
+docker push quay.io/btofel/controller-bundle:v2.0.0
+opm registry add --bundle-images quay.io/btofel/controller-bundle:v2.0.0 --database "test-registry.db"
```

Comment:
OPM sometimes might not work well in all envs. Also, are we using that only to extract the db? If yes, could we replace it by (e.g): 

```
docker create --name rh-catalog registry.redhat.io/redhat/redhat-operator-index:v4.6 "yes"
docker cp rh-catalog:/database/index.db .
```

Reply:
Why wouldn't OPM work? 
Can you fit your suggested Docker to work with this example? The current opm command is referring to the docker lines above in the README.

Reply:
In my machine that does not work for example. I need to upgrade the go 1.15.x to 1.5.x+ for that work.
You wrote that using 1.14+. So, if someone be using 1.4 that will not work really.
Also, the opm here is only used to extract the database and that is not required at all since it can be done purelly with docker.